app/controllers/operations_controller.rb

```diff
@@ -0,0 +1,61 @@
+class OperationsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_user
+  before_action :set_group
+  before_action :set_operation, only: %i[show edit update destroy]
+
+  def index; end
+
```

Comment:
- [ ] For a given category, make sure the list of transactions is presented ordered by the most recent. You can add a method in the transaction modal and use it or you can use an other approach.

eg:
```ruby
def recent_trans...
  transactions.order(created_at: :desc)
end
```

Reply:
Done! I did the modification to the show action for the groups' controller